src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -80,6 +84,7 @@ internal abstract partial class CommonSemanticQuickInfoProvider : CommonQuickInf
             foreach (var linkedDocumentId in linkedDocumentIds)
             {
                 var linkedDocument = document.Project.Solution.GetDocument(linkedDocumentId);
```

Comment:
Do we not have `GetRequiredDocument`? That seems to follow other patterns we've been doing.

Reply:
We didn't actually. Added it now.